src/components/Create/components/RewardImage/RewardImage.tsx

```diff
@@ -0,0 +1,23 @@
+import { LoadingOutlined } from '@ant-design/icons'
+import { Image, ImageProps } from 'antd'
+import { stopPropagation } from 'react-stop-propagation'
```

Comment:
**question (nonblocking)**: hmmm, is it worth adding a dep for this? :sweat-smile:

Reply:
lol - i wrote it 👀 we can remove if you dont like though - LMK